middleware/isManager.js

```diff
@@ -0,0 +1,9 @@
+const useContext = require("./useContext");
+
+module.exports = async(req, res, next) => {
+  const user = await useContext(req, res);
+  req.user = user;
+
+  if (user.isManager) next();
+  return res.status(400).send(`Invalid request !`);
```

Comment:
Maybe 403 status would be better?
 return res.status(403).send("Access denied: No permission.");

Reply:
You could also consider 401 status error, which is Unauthorised.